superset-frontend/src/dashboard/components/RefreshIntervalModal.tsx

```diff
@@ -71,17 +80,23 @@ class RefreshIntervalModal extends React.PureComponent {
 
   onSave() {
     this.props.onChange(this.state.refreshFrequency, this.props.editMode);
-    this.modalRef.current.close();
```

Comment:
could this be `this.modalRef.current?.close();`? then you wouldn't need the helper function